src/main/webapp/deletefile.html

```diff
@@ -26,42 +26,36 @@ <h2>Delete Configuration</h2>
       <a href="createfile.html">Create Configuration</a>
     </div><br>
     <div id="delete-search">
-      <form id="delete-form">
-        <span>Customer ID:</span>
-        <input list="customer-ids">
-        <datalist id="customer-ids">
-        </datalist><br>
-        <span>Account ID:</span>
-        <input list="account-ids">
-        <datalist id="account-ids">
-          <option value="Account 1">
-          <option value="Account 2">
-          <option value="Account 3">
-        </datalist><br>
+      <form onSubmit="return deleteVendor(this)">
```

Comment:
is `return deleteVendor(this)` the correct way to call this? not javascript::deleteVendor()?

Vincent might know more about this